clients/drcachesim/common/options.cpp

```diff
@@ -384,11 +384,13 @@ droption_t<bytesize_t>
                 "and the references following the simulated ones are dropped.");
 
 droption_t<std::string>
-    op_view_syntax(DROPTION_SCOPE_FRONTEND, "view_syntax", "att",
+    op_view_syntax(DROPTION_SCOPE_FRONTEND, "view_syntax", "att/arm/dr",
                    "Syntax to use for disassembly.",
                    "Specifies the syntax to use when viewing disassembled offline traces."
+                   // TODO i#4382: Add aarch64 syntax support.
                    "The option can be set to one of att (default), intel, dr and arm."
-                   "An invalid specification falls back to the default.");
+                   "An invalid specification falls back to the default, which is \"att\""
+                   "for x86, \"arm\" for ARM (32-bit), and \"dr\" for AArch64.");
```

Comment:
Nit: some users may not be clear what "dr" means. Perhaps "native dr" or "native dynamorio format"?

Reply:
+1.  Done.